OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/patches/patches.download

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# Oracle Unified Directory
+# Download patches from : https://support.oracle.com
+#
+# To build the OUD image with patches, you must download and drop the OUD 12.2.1.4.0 patch zip files (for e.g. p30851280_122140_Generic.zip) into the patches directory.
```

Comment:
Why are there two patch directories, i.e. `opatch_patch` and `patches`?

Reply:
OPatch patches are required to be applied before product patches and OPatch patches are to be applied in different way. So, there are two separate folders for OPatch patches and product patches.

Reply:
Ok, thanks for the clarification.

Reply:
Currently, README file(s) are not having anything specifically documented about patches. Until we have the README updated with patches related instructions, it would be good if you can let us have this patches.download file where its mentioned to keep downloaded patches in the directory.

Reply:
If the README has nothing documented, then these files are definitely not required either. When you have patches to apply, then you can submit a new PR that updates the README and adds these files. They do nothing except increase confusion now.

Reply:
Until README is updated, based on this "To build the OUD image with patches, you must download and drop the OUD 12.2.1.4.0 patch zip files (for e.g. p30851280_122140_Generic.zip) into the patches directory." in patches.download file, we are trying to convey that patch zip files can be kept in the folder.

Reply:
But there are no patches! Until there are patches and you can document them and point to them, get rid of all of this.